src/contracts/middleware/VoteWeigherBase.sol

```diff
@@ -16,86 +16,95 @@ import "./VoteWeigherBaseStorage.sol";
  * by the middleware for each of the quorum(s)
  * @dev
  */
-abstract contract VoteWeigherBase is VoteWeigherBaseStorage {
-    /// @notice emitted when `strategy` has been added to the array at `strategiesConsideredAndMultipliers[quorumNumber]`
-    event StrategyAddedToQuorum(uint256 indexed quorumNumber, IStrategy strategy);
+contract VoteWeigherBase is VoteWeigherBaseStorage {
+    /// @notice emitted when a new quorum is created
+    event QuorumCreated(uint8 indexed quorumNumber);
+    /// @notice emitted when `strategy` has been added to the array at `strategiesConsideredAndMultipliers[quorumNumber]` with the `multiplier`
+    event StrategyAddedToQuorum(uint8 indexed quorumNumber, IStrategy strategy, uint96 multiplier);
     /// @notice emitted when `strategy` has removed from the array at `strategiesConsideredAndMultipliers[quorumNumber]`
-    event StrategyRemovedFromQuorum(uint256 indexed quorumNumber, IStrategy strategy);
+    event StrategyRemovedFromQuorum(uint8 indexed quorumNumber, IStrategy strategy);
+    /// @notice emitted when `strategy` has its `multiplier` updated in the array at `strategiesConsideredAndMultipliers[quorumNumber]`
+    event StrategyMultiplierUpdated(uint8 indexed quorumNumber, IStrategy strategy, uint256 multiplier);
```

Comment:
noting that the events `StrategyAddedToQuorum` and `StrategyMultiplierUpdated` have different uint types for the `multiplier` field (uint96 vs. uint256). I presume we are emitting _both_ events when a new strategy is added, so think deleting the field from the `StrategyAddedToQuorum` might make sense?
ping to @gpsanant as I think he's most knowledgeable about how we're indexing this

Reply:
adding: it looks like we are *not* emitting a `StrategyMultiplierUpdated` event inside of the `_addStrategiesConsideredAndMultipliers` function. IMO we should do this and remove the `multiplier` field from the `StrategyAddedToQuorum` event, as I suggested above.
I'm open to other ideas, just feel like this is the most logical.

Reply:
@gpsanant any input on the proper event behavior here? I gave my suggestion above.

Reply:
your suggestion sounds good

Reply:
working on this now.

Reply:
@gpsanant please check out the state VoteWeigher events as of https://github.com/Layr-Labs/eigenlayer-contracts/pull/35/commits/f26ee275f77d12bf82e2b27d331e396de052c039 and resolve this thread if you're satisfied